src/navigation/NavigationTypes.ts

```diff
@@ -22,15 +22,15 @@ interface VocabularyDetailExerciseParams extends Omit<ExerciseParams, 'disciplin
   disciplineId: number | null
 }
 
-export interface ExercisesParams extends Omit<ExerciseParams, 'documents' | 'closeExerciseAction'> {
+export interface ExercisesParams extends Omit<ExerciseParams, 'vocabularyItems' | 'closeExerciseAction'> {
```

Comment:
I would suggest, to rename ExerciseParams in this line. Having the same name for two Interfaces (see line 12) was confusing for me while refactoring.